btk/draw_blends.py

```diff
@@ -373,93 +430,25 @@ def render_single(self, entry, cutout, band):
         if self.verbose:
             print("Draw isolated object")
 
-        galaxy_builder = descwl.model.GalaxyBuilder(
-            cutout, no_disk=False, no_bulge=False, no_agn=False, verbose_model=False
-        )
-
+        pix_stamp_size = np.int(self.stamp_size / survey.pixel_scale)
         try:
-            galaxy = galaxy_builder.from_catalog(entry, entry["ra"], entry["dec"], band)
-            iso_render_engine = descwl.render.Engine(
-                survey=cutout,
-                min_snr=self.min_snr,
-                truncate_radius=30,
-                no_margin=False,
-                verbose_render=False,
+            gal = get_catsim_galaxy(entry, filt, survey)
+            gal_conv = galsim.convolve.Convolution(gal, psf)
+            return gal_conv.drawImage(
+                nx=pix_stamp_size, ny=pix_stamp_size, scale=survey.pixel_scale
             )
-            iso_render_engine.render_galaxy(
-                galaxy,
-                variations_x=None,
-                variations_s=None,
-                variations_g=None,
-                no_fisher=True,
-                no_analysis=True,
-            )
-
-            return cutout.image
 
-        except descwl.render.SourceNotVisible:
+        except SourceNotVisible:
```

Comment:
Is this really useful ? I do not think the exception will be raised by our code.

Reply:
I guess this could happen if the catalog provided by the user has galaxies with 0 fluxes, but I guess then the user could remove them from the catalog. I'll get rid of this

Reply:
Will be raised by `get_catsim_galaxy` if  
```python
disk_flux + bulge_flux + agn_flux == 0
```

Reply:
thinking about it more, maybe I'll keep it so the user is aware that they passed in an object with zero-flux if they do.